cmd/bnsd/x/username/model_test.go

```diff
@@ -75,6 +78,36 @@ func TestBlockchainAddressValidation(t *testing.T) {
 	}
 }
 
+func TestQueryByOwner(t *testing.T) {
+	var retrievedTokens []Token
+
+	db := store.MemStore()
+	migration.MustInitPkg(db, "username")
+	username := Username("alice*iov")
+
+	token := Token{
+		Metadata: &weave.Metadata{Schema: 1},
+		Targets: []BlockchainAddress{
+			{BlockchainID: "blockchain", Address: []byte("123456789")},
```

Comment:
`Address:` should be a string now

Reply:
it should, but it isn't. It's not the scope of this PR, so this has to be done when the appropriate changes are merged I guess.

Reply:
This was changes in the `launch` branch already: https://github.com/iov-one/weave/pull/865 So it will need manual merging now.

Reply:
Ah, it's no problem. I will cherry-pick and merge. It wasn't marked as launch so I kind of missed it.